src/scripts/main.js

```
@@ -1,7 +1,45 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  // Присвоєння або створення елементів
+  const body = document.body;
+  const box = document.createElement('div');
+  const titleBox = document.createElement('h2');
+  const descriptionBox = document.createElement('p');
+
+  // Додавання класу
+  box.classList.add('notification');
+
+  // Фарбування box
+  if (type === 'success') {
+    box.classList.add('success');
+  } else if (type === 'warning') {
+    box.classList.add('warning');
+  } else {
+    box.classList.add('error');
```

Type handling currently maps unknown values to `error` via the `else` branch. Checklist item 3.5 expects you to validate and apply only the three supported types (`success`, `error`, `warning`) and ensure the created element includes the corresponding class that exactly matches the `type` value. Consider validating `type` with a whitelist and only adding the class when it's valid, or explicitly warning when an unknown type is passed. Example approaches:
- Use `const allowed = ['success','error','warning']; if (allowed.includes(type)) box.classList.add(type); else { console.warn('Unknown notification type', type); box.classList.add('error'); }`
This makes behavior explicit and prevents silent fallbacks.